src/components/Preview.css

```diff
@@ -74,6 +117,33 @@
   border-radius: 3px;
 }
 
+.directional-icon {
+  width: 100px;
+  height: 100px;
+  margin: 5px;
+  position: relative;
+}
+
+.Frame-timeline {
+  width: 95vw;
+  display: flex;
+  overflow-x: scroll;
+  margin-top: 5vw;
+}
```

Comment:
you can make sure there is a little bit of space on either side of the `.Frame-timeline` by changing the `width` and `margin-top` styles, i.e:
```
.Frame-timeline {
    width: calc(95vw - 40px - 2vw);
    display: flex;
    overflow-x: scroll;
    margin: 5vw 2vw;
}
```